### PR TITLE
CT 116: Add VET TEC logo to Comparison Tool VET TEC Profile page

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a Comparison Tool user, I need to see the VET TEC logo on the VET TEC Profile Page so that I have a consistent experience while viewing VET TEC related materials.
[Project Management and Software Development with GitHub | ZenHub](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19184)

## Testing done
Local Manual, Unit, and e2e test pass

## Screenshots
Screen Shot 2019-07-18 at 2 50 05 PM
Screen Shot 2019-07-18 at 2 50 24 PM

## Acceptance criteria
- [x] The VET TEC Logo is not displayed when the Institution is not a VET TEC provider (as indicated by the Facility Code in the WEAMS.csv file).
- [x] The VET TEC logo is displayed in the top right corner of the page when the Institution is a VET TEC provider (as indicated by the Facility Code in the WEAMS.csv file).
- [x] The VET TEC logo is 85px high.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Similar to your other PR, can you use the shortened paths for your imports. Please move your inline style to a css class and refer to the class.

**Review by zurbergram — CHANGES_REQUESTED** [review]
Need UI/UX review on mobile version design

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 1]
```diff
@@ -1,4 +1,24 @@
 .gi-app .profile-page {
+  .vettec-logo-container {
+    text-align: right;
+  }
+  .vettec-logo {
+    height: 85px;
+    width: 179px;
+    padding-bottom: 8px;
```
Comment (cvalarida) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: While I think it makes sense to have `width` and `height` in `px`, I typically prefer paddings to be in `em` myself. I don't remember exactly why I started, but I think it had something to do with zooming and scaling up the font size. Something something mobile devices too?

I definitely won't say this is wrong like this or anything, but I generally recommend `em`. Maybe try zooming and looking at it on a mobile device to see if it still looks like you expect it to.

**Review by cvalarida — APPROVED** [review]
It's probably worth double-checking this looks like you want it when zoomed in, but if so, :shipit: :+1:

**Pull request merged**